### DISCUSSION TITLE
How to disable reference line and dots on hover?

### DISCUSSION THREAD

**Discussion by stefan-girlich** [recharts/recharts#4444] [Q&A] [answered]
I want to disable the default dots and reference line displayed on hover together with the tooltip:

![Screenshot 2024-04-18 at 17 52 16](https://github.com/recharts/recharts/assets/1755315/a7e9e06d-c5f8-419a-a547-dcd422ad854d)
![Screenshot 2024-04-18 at 17 48 47](https://github.com/recharts/recharts/assets/1755315/bdb6eae5-d073-4221-a815-faa109ecb1da)

I'm using recharts v2.12.5.

Any idea? I bet there's a property on `<Tooltip />` that does exactly that; but I can't figure out which one.

**Comment by ckifer** [comment, marked as answer]
its on `Area` - `activeDot`

https://recharts.org/en-US/api/Area#activeDot

`activeDot={false}` should do it

**Reply by stefan-girlich** [reply]
Thanks, it worked! Would you also know how to disable the vertical line?

**Reply by ckifer** [reply]
thats the cursor on Tooltip

So:

`<Tooltip cursor={false} />`

**Reply by stefan-girlich** [reply]
This only works when not using a custom tooltip; not sure if this is a bug or expected.

**Reply by ckifer** [reply]
Hmm, it works here [![Edit tooltip-with-customized-content (forked)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/tooltip-with-customized-content-forked-mdmlxm)

Will try it with an Area chart real quick one sec...

**Reply by ckifer** [reply]
[![Edit tooltip-with-customized-content (forked)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/tooltip-with-customized-content-forked-y5dq57)

Still works with an AreaChart. So not sure whats going on, can you try to reproduce in a sandbox?

**Reply by stefan-girlich** [reply]
My bad - I accidentally added `cursor={false}` to the custom component itself which is also a valid property.
Sorry for the confusion and thanks for the very quick support; I appreciate it.

**Reply by ckifer** [reply]
No problem!
